### ISSUE TITLE
Support bookmark for source code of  java maven dependencies

### ISSUE THREAD

**Issue by kchenzhi** [alefragnani/vscode-bookmarks#307]
Hi, I use vscode-bookmarks in my java project and it performs very well. 
But when I want to bookmark maven's dependency source code, there will be some problems:
![Screen Shot 2020-06-12 at 10 07 41 AM](https://user-images.githubusercontent.com/28884725/84457228-b16bf100-ac94-11ea-89c3-1aa8b42fd993.png)
The icon of the bookmark is appare, but the BOOKMARKS:EXPLORER is empty, I can't use bookmark to jump to the position of the code.

**Comment by alefragnani** [comment]
Hi @kchenzhi ,

I think the problem occurs because these `.class` files, in fact, are not _files in the file system_. Instead, are stored inside the java/maven packages, and the Java/Maven extension (I’m not sure which one you use) support opening/navigating/displaying these _files_.

If that’s the case, this could be achieved only after the **Remote Development** (#230) is released. I have to change how files are handled by the extension, to support these scenarios.

Thanks for reporting

**Comment by kchenzhi** [comment]
Yes, I think it's the problem of `.class` files in maven packages.
 This is the major reason why I am not using vscode to develop java.
Anticipate for **Remote Development**  release, thank you for your job.

**Comment by alefragnani** [comment]
I don't use Java so often (last time was a year ago) but I remember using VS Code only for quick navigation purposes. The Java extension was still too limited and heavy, compared to Eclipse (at least for my needs). But I guess the same issue could happened for similar languages, like C#.

**Comment by alefragnani** [comment]
Hi ,

With a new year starting, it is the moment to review the extension’s backlog in order to keep it healthy and focused.

I'm closing this feature request because it didn’t receive enough upvotes from the community. 

Still a big Thank You to you for taking the time to create this issue!

Thanks for your understanding
